src/bash_src/analysis/singleAnalysis/mergeSingleMethAnalysis/mergeSingleMethAnalysis_help.txt

```diff
@@ -5,17 +5,18 @@ Usage:
     mergeSingleMethAnalysis.sh [options] -- PREFIX CHR_NUM TOTAL_PROC
```

Comment:
We should document what the mandatory arguments are just like we do with the options. Also should we use "phenoname" rather than "prefix"? It is what we used to do, and I don't recall why we changed it. Phenoname seems more descriptive to me.

Reply:
So that's been done for the functions the average user will interact with. However, for the other functions I don't see the need. 

Although this wasn't strictly necessary, I didn't feel comfortable referring to what actually is a prefix as phenotype when writing the bash wrappers, I just found it ambiguous and confusing. However, I do agree that, for the functions the average user interacts with, we should use the word phenotype in the help file and that is indeed the case.

Reply:
On a second thought, for consistency as far as documentation goes I agree with you. SO I will go ahead and replace the `PREFIX` for `PHENO` in the rest of help files.

Reply:
Taken care of.